.github/workflows/release_prep.sh

```diff
@@ -15,6 +15,14 @@ ARCHIVE="rules_mylang-$TAG.tar.gz"
 git archive --format=tar --prefix=${PREFIX}/ ${TAG} | gzip > $ARCHIVE
 SHA=$(shasum -a 256 $ARCHIVE | awk '{print $1}')
 
+# Add generated API docs to the release, see https://github.com/bazelbuild/bazel-central-registry/issues/5593
+docs="$(mktemp -d)"; targets="$(mktemp)"
+bazel --output_base="$docs" query --output=label --output_file="$targets" 'kind("starlark_doc_extract rule", //...)'
+bazel --output_base="$docs" build --target_pattern_file="$targets" --remote_download_regex='.*doc_extract\.binaryproto'
+tar --create --auto-compress \
+    --directory "$(bazel --output_base="$docs" info bazel-bin)" \
```

Comment:
Could this use `cuqery --output=files` instead? This is a bit wonky otherwise

Reply:
yes I have inspected the archives, there aren't any other output files present with this technique.

Reply:
Do you prefer https://github.com/bazel-contrib/bazel_features/pull/116/files#diff-bcaf23ede093ab9c0f7a9579787d5e631a22c42804480b718eae5a22f96edeb0R14-R18 ?